src/deck/drivers/src/audio_deck.c

```diff
@@ -259,14 +188,9 @@ void fill_tx_buffer() {
 	param_buffer_uint16[N_MOTORS + 1] = max_freq;
 	param_buffer_uint16[N_MOTORS + 2] = delta_freq;
 	param_buffer_uint16[N_MOTORS + 3] = n_average;
+	param_buffer_uint16[N_MOTORS + 4] = filter_prop_enable;
+	param_buffer_uint16[N_MOTORS + 5] = filter_snr_enable;
 
-	uint16_t enables = (filter_propellers_enable << 8) | filter_snr_enable;
```

Comment:
We didn't gain anything from splitting the uint16 into two uint8, since anyways we have to send a very long buffer in exchange for the audio data.